geode-core/src/main/java/org/apache/geode/cache/client/internal/AutoConnectionSourceImpl.java

```diff
@@ -253,12 +263,15 @@ protected void updateLocatorList(LocatorListResponse response) {
     List<ServerLocation> locatorResponse = response.getLocators();
 
     List<InetSocketAddress> newLocators = new ArrayList<InetSocketAddress>(locatorResponse.size());
+    List<InetSocketAddress> activeLocators =
```

Comment:
I think "active" is another adjective where you should consider using "online" instead.